app/app.py

```diff
@@ -21,3 +29,21 @@ def home_page():
     Create the home page for the website.
     """
     return "<p>This is the home page! Make sure the other routes are in different files!</p>"
+
+
+@app.route("/data/average", methods=["GET"])
```

Comment:
maybe a route like `/<name>/average` might be better?

Reply:
this is honestly my bad, i totally forgot these were get routes. all the params should be in the route name.